snapshots/climate/2025-02-12/sst.py

```diff
@@ -0,0 +1,90 @@
+"""Script to create a snapshot of dataset."""
+
+from io import StringIO
+from pathlib import Path
+
+import click
+import pandas as pd
+import requests
+from owid.datautils.io import df_to_file
+
+from etl.snapshot import Snapshot
+
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
+BASE_URL_ONI = "https://www.cpc.ncep.noaa.gov/data/indices/oni.ascii.txt"
+BASE_URL = "https://www.cpc.ncep.noaa.gov/data/indices/sstoi.indices"
+
+
+def season_to_month(season: str) -> int:
+    """
+    Convert the season string to the corresponding month.
+
+    Parameters:
+    season (str): The season string (e.g., "DJF").
+
+    Returns:
+    int: The corresponding month (1-12).
+    """
+    season_to_month_map = {
+        "DJF": 2,  # December-January-February -> January
```

Comment:
Maybe clarify why you are mapping to the latter month instead of the middle one (which is what I understand from the comment on the right).